app/blueprints/assessments/templates/assessor_tasklist.html

```diff
@@ -102,6 +103,15 @@
         {% endif %}
     </span>
     </div>
+
+    <div class="gov-uk-table__cell--align-right">
+        <span class="left govuk-!-padding-right-4">
+            {% if g.access_controller.is_lead_assessor and state.fund_short_name|upper =="COF-EOI" %}
+            {{ link_to_download_contract_docs(csrf_token, application_id) }}
+        {% endif %}
```

Comment:
Can we just line these if statements up 🙏 otherwise nice work! 👍 

```suggestion
            {{ link_to_download_contract_docs(csrf_token, application_id) }}
            {% endif %}
```